internal/server/apimigrator.go

```diff
@@ -23,26 +23,28 @@ type apiMigration struct {
 	method          string
 	path            string
 	version         string
-	redirect        string
+	rename          string
 	requestRewrite  func(c *gin.Context)
 	responseRewrite func(c *gin.Context)
 }
 
-func addRedirect(a *API, method, path, newPath, version string) {
-	// update any existing migrations with the legacy path
-	for i, mig := range a.migrations {
-		if len(mig.redirect) == 0 && mig.path == path {
+// TODO: rename to addRename
```

Comment:
Is there something blocking renaming this function in this PR?

Reply:
Nothing blocking. It is going to increase the size of the diff quite a bit, so I wanted to avoid it initially to keep the focus on the other changes. It should be an easy "refactor -> rename" in GoLand in a follow up